### PR TITLE
Implementing feedback from meeting 04.07.2024

### PR DESCRIPTION
**Pull request by mkuehbach**
@srmnitc  DON'T MERGE IT YET, as discussed preparation branch for implementing final touches!

- [x] IFES interface for RRNG, APT, and POS, tested
- [x] Test has to be reverted

### CODE REVIEW

**Comment by mkuehbach** [comment]
We have a few unnecessarily large files in the repository we should clean these, see the following discussion on how to do this:
https://github.com/FAIRmat-NFDI/nexus_definitions/issues/237

I tested it locally:
```
.../CompositionSpace$ git count-objects -vH
count: 0
size: 0 bytes
in-pack: 869
packs: 1
size-pack: 218.70 MiB
prune-packable: 0
garbage: 0
size-garbage: 0 bytes
.../CompositionSpace$ git filter-repo --invert-paths --path-glob "*.pos" --path-glob "*.zip" --path-glob "*.tar.gz" --path-glob "*.nxs" --path-glob "*.h5"
Parsed 156 commits
New history written in 0.05 seconds; now repacking/cleaning...
Repacking your repo and cleaning out old unneeded objects
HEAD is now at 77b522c Update environment.yml
Enumerating objects: 857, done.
Counting objects: 100% (857/857), done.
Delta compression using up to 32 threads
Compressing objects: 100% (317/317), done.
Writing objects: 100% (857/857), done.
Total 857 (delta 524), reused 856 (delta 523), pack-reused 0
Completely finished after 0.16 seconds.
.../CompositionSpace$ git count-objects -vH
count: 0
size: 0 bytes
in-pack: 857
packs: 1
size-pack: 2.92 MiB
prune-packable: 0
garbage: 0
size-garbage: 0 bytes
```

Instead these files should be pointed to via another place, e.g. Zenodo repository!

**Comment by mkuehbach** [comment]
@srmnitc read to review :)

File: .bumpversion.cfg
Comment (srmnitc) [inline .bumpversion.cfg:1]: Is there any reason why this is deleted. I use bumpversion for version tracking.

**Review by srmnitc — APPROVED** [review]
lgtm

the only issue is why bumpversion file was removed. Please let me know if there was a specific reason for this. If not, I will add it back. Thanks!

**Comment by srmnitc** [comment]
@mkuehbach thanks for all your efforts. I am going to merge this in, and do a bit of house keeping. After that I will release a new version, update conda-recipe and we should be good to go.

File: .bumpversion.cfg
Comment (mkuehbach) [reply .bumpversion.cfg:1]: No there wasnt I was unsure whether this file is needed, internally we typically do version management with setuptools for which then the pyproject.toml alone suffices but indeed if you need this file sure it should stay.

**Pull request merged**